packages/ui/src/Typography.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Body } from './Body';
+import { Detail } from './Detail';
+import { H1 } from './H1';
+import { H2 } from './H2';
+import { H3 } from './H3';
+import { H4 } from './H4';
+import { Large } from './Large';
+import { P } from './P';
+import { Small } from './Small';
+import { Strong } from './Strong';
+import { Technical } from './Technical';
```

Comment:
nitpick: the directory structure is unnecessary huge. Can we make it all exported from one?

Reply:
Will fix per comment above